tests/foreman/cli/test_abrt.py

```diff
@@ -24,7 +24,7 @@ def test_abrt_1(self):
 
         """
 
-    def test_abrt_2(self):
+    def test_positive_create_multiple_reports(self):
```

Comment:
Does this create a ``crash`` report as well as `test_positive_create_crash_report`? If so, perhaps we want to rename this to `test_positive_create_reports`?